util/solder/solder.axi_id_serialize.sv.tpl

```diff
@@ -1,5 +1,4 @@
   axi_id_serialize #(
-    .AtopSupport (1),
```

Comment:
This can be reverted since i cleaned up the `hw/ips/future` IPs related to AXI which are now found in the `vendor/pulp_platform_axi`. I added the `AtopSupport` parameter as a [patch ](https://github.com/pulp-platform/snitch/blob/master/hw/vendor/patches/pulp_platform_axi/0007-vendor-add-AtopSupport-parameter-to-axi_id_serialize.patch) to `pulp_platform_axi`

Reply:
Thanks for the review, this is reverted :+1: I suggest we wait until tomorrow and let the scheduled FPGA test run before merging. I also had to fix sth related to the nightly test in [d3f91ca](https://github.com/pulp-platform/snitch/pull/389/commits/d3f91cacbe74ffbfd0e0598652910a5a6e3f1cc9)